pkg/cmd/roachtest/cluster_test.go

```diff
@@ -282,8 +282,14 @@ func TestAWSMachineType(t *testing.T) {
 				}
 				testCases = append(testCases, machineTypeTestCase{i, mem, false, arch,
 					fmt.Sprintf("%s.%s", family, xlarge(i)), arch})
-				testCases = append(testCases, machineTypeTestCase{i, mem, true, arch,
-					fmt.Sprintf("%sd.%s", family, xlarge(i)), arch})
+				if family != "c6a" {
```

Comment:
Nits: might be clearer to avoid the double negative on the `else`. And maybe just set the "expectedMachineType" conditionally, to make it clearer that the rest remains the same.

Reply:
Yep, will simplify.